tests/blockchain/test_blockchain_transactions.py

```diff
@@ -890,6 +890,19 @@ async def test_assert_seconds_relative(
             full_node_1.blockchain, invalid_new_blocks[-1], expected_error=Err.ASSERT_SECONDS_RELATIVE_FAILED
         )
 
+        # we compare the timestamp against the previous transaction block, so in
```

Comment:
Is this really part of the patch? I don't immediately see the connection.

Reply:
yes, this is in fact required for the new consensus rules. This test used to only run pre soft-fork2.
One change in soft-fork2 was that relative time conditions now compare against the timestamp of the previous transaction block (as opposed to its own timestamp). This made it aligned with the relative height conditions, which compared against the height of the previous transaction block.

This test is specifically trying to trigger a relative time condition failure by adjusting the timestamp of the block that also has the condition. The previous way this test did this doesn't work anymore because the *current* block's timestamp is no longer used, but the previous transaction block's.

This fix inserts one more block that allows use to bump the timestamp of the block that's used for the conditions